ddl/cancel_test.go

```diff
@@ -227,17 +229,24 @@ func TestCancel(t *testing.T) {
 	}
 
 	// Change some configurations.
-	ddl.ReorgWaitTimeout = 10 * time.Microsecond
+	ddl.ReorgWaitTimeout = 10 * time.Millisecond
 	tk.MustExec("set @@global.tidb_ddl_reorg_batch_size = 8")
+	tk.MustExec("set @@global.tidb_ddl_reorg_worker_cnt = 1")
+	tk = testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/ddl/mockBackfillSlow", "return"))
+	defer func() {
+		require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/ddl/mockBackfillSlow"))
+	}()
 
 	hook := &ddl.TestDDLCallback{Do: dom}
-	i := 0
+	i := atomicutil.NewInt64(0)
```

Comment:
Why change it to atomic?

Reply:
Idk, I just cherry-picked this code, maybe @hawkingrei could give some explanation?

Reply:
IMO，`MustExec` will block ddl stmt until it ends and it will not have data race.

Maybe use atomic in later PR about `concurrency ddl` is better.

Reply:
ok, actually I think there will be no data race even concurrent ddl, maybe I am wrong.